packages/shared/src/components/sidebar/Sidebar.tsx

```diff
@@ -63,6 +58,7 @@ export default function Sidebar({
     loadedSettings,
     optOutWeeklyGoal,
   } = useContext(SettingsContext);
+  const { openSquadBetaModal } = useCreateSquadModal();
```

Comment:
Love the cleanup here! ✨

Reply:
Ah yes, was needed for the "generic" open.
This won't interfere with the addition where you hide the sidebar? (Think not as it's only called from a notification)